.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
I *really* dislike this setting.  I've been bitten in the past with allowing git to do stuff behind your back, and worse than that -- when things get messed up it's difficult to sort it out when other people are not even aware of the problem.